docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+  pqai_reranker_api:
+    container_name: dev_pqai_reranker_api
```

Comment:
Please remove the `dev_` prefix and change the filename to `docker-compose.prod.yml` (README might also require updation)